types/react-onsenui/index.d.ts

```diff
@@ -3,11 +3,17 @@
 // Definitions by: Ozytis <https://ozytis.fr>,
 //                 Salim <https://github.com/salim7>,
 //                 Jemmyw <https://github.com/jemmyw>
+//                 Mikael Lirbank <https://github.com/lirbank>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 // TypeScript Version: 2.8
 
 import * as React from 'react';
-import Component = React.Component;
+
+export type Optional<T> = { [P in keyof T]?: T[P] };
```

Comment:
isn't this provided by default typescript types as `Partial`?

Reply:
Ahh, you're right! Not sure what I was thinking. Fixed now.